lib/src/controls/navigation/navigation_view/pane_items.dart

```diff
@@ -679,20 +685,73 @@ class __PaneItemExpanderState extends State<_PaneItemExpander>
     return body.pane?.effectiveIndexOf(widget.item) ?? 0;
   }
 
-  void toggleOpen() {
+  void toggleOpen({bool doFlyout = true}) {
     setState(() => _open = !_open);
 
-    PageStorage.of(context)?.writeState(
+    PageStorage.of(context).writeState(
       context,
       _open,
       identifier: 'paneItemExpanderOpen$index',
     );
-    if (useFlyout) {
-      flyoutController.toggle();
-    }
     if (_open) {
+      if (useFlyout && doFlyout) {
+        final body = InheritedNavigationView.of(context);
+        final displayMode = body.displayMode;
+        final navigationTheme = NavigationPaneTheme.of(context);
+
+        flyoutController.showFlyout(
+          placementMode: displayMode == PaneDisplayMode.compact
+              ? FlyoutPlacementMode.right
+              : FlyoutPlacementMode.bottomCenter,
+          builder: (context) {
+            return MenuFlyout(
+              items: widget.items.map<MenuFlyoutItemBase>((item) {
+                if (item is PaneItem) {
+                  return _PaneItemExpanderMenuItem(
+                    item: item,
+                    onPressed: () {
+                      widget.onItemPressed?.call(item);
+                      Navigator.pop(context);
+                    },
+                    isSelected: body.pane!.isSelected(item),
+                  );
+                } else if (item is PaneItemSeparator) {
+                  return const MenuFlyoutSeparator();
+                } else if (item is PaneItemHeader) {
+                  return MenuFlyoutItemBuilder(builder: (context) {
+                    return Container(
+                      padding: const EdgeInsets.symmetric(
+                        horizontal: 10.0,
+                        vertical: 8.0,
+                      ),
+                      margin: const EdgeInsetsDirectional.only(bottom: 4.0),
+                      child: DefaultTextStyle(
+                        style: navigationTheme.itemHeaderTextStyle ??
+                            const TextStyle(),
```

Comment:
Is this actually needed? As in you're not allowed to pass null to it

Reply:
Yeah, `DefaultTextStyle.style` doesn't allow a null style